lib/klarna/payouts.rb

```diff
@@ -0,0 +1,11 @@
+module Klarna
+  class Payouts < Client
+    def payout(payment_reference)
+      do_request(:get, "/settlements/v1/payouts/#{payment_reference}")
+    end
+
+    def payout_summary(data)
+      do_request :get, "/settlements/v1/payouts/summary?start_date=#{data[:start_date]}&end_date=#{data[:end_date]}"
+    end
```

Comment:
Can we use keyword arguments instead of accessing the hash as we explicitly use `start_date` and `end_date` keys of the `data` hash anyway, ie:

```suggestion
    def payout_summary(start_date:, end_date:)
      do_request :get, "/settlements/v1/payouts/summary?start_date=#{start_date}&end_date=#{end_date}"
    end
```